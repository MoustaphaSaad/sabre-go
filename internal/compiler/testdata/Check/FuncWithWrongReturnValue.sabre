package main

func foo() bool {
    return 1
}
