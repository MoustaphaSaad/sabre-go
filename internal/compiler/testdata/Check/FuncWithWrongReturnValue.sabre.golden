>> 	    return "1"
>> 	           ^^^ 
Error[internal/compiler/testdata/Check/FuncWithWrongReturnValue.sabre:4:12]: incorrect return type 'string', expected 'int'
